cmd/internal/apidump/apidump.go

```diff
@@ -80,6 +84,17 @@ var Cmd = &cobra.Command{
 			outFlag.AkitaURI = &uri
 		}
 
+		// If --collection was given, set rest.domain and environment config (if given)
+		if postmanCollectionID != "" {
+			if postmanEnvironment != "" {
+				env := strings.ToUpper(postmanEnvironment)
+
+				cfg.WritePostmanEnvironment("default", env)
+			}
+
+			rest.Domain = "staging.akita.software"
```

Comment:
See my comment in https://github.com/akitasoftware/akita-cli/pull/214, I think to do this right we need to pull the environment -> domain logic up into the root command.